app/jobs/add_works_to_collections_job.rb

```diff
@@ -7,6 +7,9 @@ def perform(record_data:)
     work = record_data[:model].constantize.where(identifier: record_data[:identifier]).first
     collection = Collection.where(identifier: record_data[:parents]).first
 
+    return if !work.present? && !collection.present?
+    return if work.member_of_collection_ids.include?(collection.id)
+
```

Comment:
confirmed that this is working. When the id isn't included it created the relationship: 

![image](https://github.com/scientist-softserv/adventist-dl/assets/10081604/d8ede94d-f556-44f7-8497-aaf81aa0d611)


![image](https://github.com/scientist-softserv/adventist-dl/assets/10081604/9688e89b-ad2a-43ac-8d5b-4b2e5a949d83)